content/lua/concepts/tables/terms/move/move.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'move()'
+Description: 'Copies elements from one table to another.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Tables'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`.move()`** function copies the specified indices' values from the source table into the provided starting index of the destination table.
+
+## Syntax
+
+```lua
+table.move(a1, f, e, t, a2)
+```
+
+`.move()` function expects five arguments where `a1` is the source table and `a2` is the destination table.
+
+`f` means _from_ (First index within `a1`).
+
+`e` means _end_ (Last index within `a1`).
+
+`t` means _to_ (First index within `a2`).
+
+## Example
+
+Consider the following tables:
+
+```lua
+local Table1 = {"🍎", "🍌", "🍇", "🍓", "🍉" }
+
+local Table2 = {}
```

Comment:
```suggestion
local Table1 = {"🍎", "🍌", "🍇", "🍓", "🍉" }

local Table2 = {"🍎", "🍌"}
```

Reply:
I think the operation might be more obvious if the move is happening to a table that has some elements